src/factorial.js

```diff
@@ -0,0 +1,7 @@
+export default function factorial(num){
+  let total = 1
+  for (var i = 1; i <= num; i++) {
+    total = total * i
+  }
+  return total
```

Comment:
i cant remember if this is part of the spec or not, however it is good practice to have error checking for factorial. something like `if (n < 0){
    return "Please enter a positive integer"
  }`